setup.py

```diff
@@ -1,18 +1,43 @@
 #!/usr/bin/python3
 import setuptools
+import sys
 import PyInstaller.__main__ as pim
 
-pim.run([
-	"--name=DodgeToDrums",
-	"--onefile",
-	"--windowed",
-	"--clean",
-	"--exclude-module=tkinter",
-	"--exclude-module=PyQt4",
-	"--exclude-module=PyQt5",
-	"--exclude-module=numpy",
-	"--hidden-import=packaging.requirements",
-#	"--icon=icon.ico",
-#	"--add-data=./data/",
-	"main.py"
-	])
+DEFAULT_PROFILE = "dist"
+
+if len(sys.argv) > 0:
+	PROFILE = sys.argv[1]
+else:
+	PROFILE = DEFAULT_PROFILE
```

Comment:
I think the DEFAULT_PROFILE variable is unnecessary since it only gets used once. If you want to make it clear that this is the default value, just make a comment stating that.

Reply:
Allright, I'll implement that